WordPress/src/test/java/org/wordpress/android/ui/reader/discover/ReaderDiscoverViewModelTest.kt

```diff
@@ -187,6 +193,43 @@ class ReaderDiscoverViewModelTest {
                 assertThat(contentUiState.cards.first()).isInstanceOf(ReaderPostUiState::class.java)
             }
 
+    @Test
+    fun `if WelcomeBannerCard exists then ReaderWelcomeBannerCardUiState will be present in the ContentUiState`() =
```

Comment:
I'm wondering if we should add a test to make sure we don't show the ContentUiState when the only available data is the WelcomeCard -> in other words, that we don't show the welcome card when the discover feed is empty. Wdyt?

Reply:
Test added in 58a61ca

Reply:
Awesome, thanks!